### ISSUE TITLE
Description for "Create time bound request" needs cleanup

### ISSUE THREAD

**Issue by mwaterous** [opsramp/openapi-specs#58]
https://github.com/opsramp/openapi-specs/blob/a5cb1095590f34d70add7dcb0bb449235d0253aa/v2/opsramp-tickets.v2.yaml#L13757

The description includes a sample URL, which is a planned feature for a future release of the documentation site. It doesn't make sense to include this on some endpoints but not all, recommend removing this from the description until it's a built-in feature.

Additionally the formatting is off here, there's a table that's not displaying correctly because it looks like the whole description was imported as a string instead of converted to multiline and markdown.

Are the inline samples absolutely necessary on this? If the properties have good descriptions attached that information might be redundant?

**Comment by rshera01** [comment]
This I will have a look and this too is a content cleanup

**Comment by mwaterous** [comment]
What are the payloads? We can use the property's description field to explain any ambiguity but the content/type should reflect the type of file and the property type can be set to string. Ultimately the solution needs to be valid OpenAPI so we're not introducing any technical debt trying to work around other solutions.

**Comment by rshera01** [comment]
Yes. the description is not needed and it becomes self explanatory when described at the Attribute level. "Content cleanup" was intended to move out from description to the properties.
Since, in the current mechanism, we cannot render , the enum values to let users know on all supported values, hence all of this ambiguous explanation in place. Such descriptions have been very much randomly put in the API docs.

**Comment by mwaterous** [comment]
Please disregard that comment in the context of this issue, I had multiple issues open across many tabs earlier and posted that comment to the wrong issue. Apologies for the confusion, I'll move it to the right issue.
